ethereum/env/.env.snaxchain.testnet

```diff
@@ -0,0 +1,21 @@
+# SnaxChain testnet env
+# Rename to .env
+
+# Common config for forge deployment
+RPC_URL="https://rpc-snaxchain-s50q0kjngn.t.conduit.xyz/"
+FORGE_ARGS="--legacy"
+
+# Wormhole Core Migrations
+INIT_SIGNERS=["0x13947Bd48b18E53fdAeEe77F3473391aC727C638"]
+INIT_CHAIN_ID=43
+INIT_GOV_CHAIN_ID=0x1
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+INIT_EVM_CHAIN_ID=2192
+
+# Bridge Migrations
+BRIDGE_INIT_CHAIN_ID=43
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+BRIDGE_INIT_FINALITY=1
+# TODO: This exists. Is this what we should use?
+BRIDGE_INIT_WETH=0x4200000000000000000000000000000000000006
```

Comment:
yes, this is the wrapped custom gas address for OP Stack chians. https://docs.optimism.io/stack/protocol/features/custom-gas-token#what-is-the-wrapped-erc-20-gas-token